MELA/interface/TCouplingsBase.hh

```diff
@@ -197,6 +197,93 @@ namespace{
 
     SIZE_AZff
   };
+  enum CouplingIndex_Smeft{
```

Comment:
'SMEFT' everywhere - let's avoid making the user think about capital vs small case letter differences.